.drone.yml

```diff
@@ -0,0 +1,62 @@
+kind: pipeline
```

Comment:
I think the drone file "should" also be added to the .eslintignore file so that it doesn't get bundled in the release?

Reply:
What does this mean? I don't see how the Drone file is related to EsLint, and further, how EsLint is responsible for the release bundle.
I don't see explicit exclusion of the .drone.yml in other projects either.

Reply:
oh, right, I meant `.npmignore` :smile: Sorry for the confusion.